Update README.md, changed ROS2 to ROS 2

IrinaTerebiznik opened the pull request:
Changed  every "ROS2"  that is not a command nor a URL to "ROS 2", as it is properly written that way.

IrinaTerebiznik committed:
Update README.md, changed ROS2 to ROS 2

FlorGrosso commented on inorbit_republisher/README.md:
```diff
@@ -148,7 +148,7 @@ This mapping option transforms the entire ROS message to a JSON string.
 
 * `mapping_type`: this mapping option should be set to `serialize`.
-* `fields`: (optional) a set of first level fields or keys to keep. If not provided, all fields are kept. For example, using the following mapping option for serializing 4 [NavSatFix](https://docs.ros2.org/latest/api/sensor_msgs/msg/NavSatFix.html) fields:
+* `fields`: (optional) a set of first level fields or keys to keep. If not provided, all fields are kept. For example, using the following mapping option for serializing 4 [NavSatFix](https://docs.ROS 2.org/latest/api/sensor_msgs/msg/NavSatFix.html) fields:
```
@IrinaTerebiznik hi! Please note that here you will be breaking the URL with this change. It should still be `ros2`

FlorGrosso commented on inorbit_republisher/README.md:
```diff
@@ -173,21 +173,21 @@ These values will be published as latched and delivered only once every time a s
 docker run -ti --rm \
-  --workdir /root/ros2_ws/ \
-  -v .:/root/ros2_ws/src/inorbit_republisher \
+  --workdir /root/ROS 2_ws/ \
```
Same here

FlorGrosso commented on inorbit_republisher/README.md:
```diff
@@ -173,21 +173,21 @@ These values will be published as latched and delivered only once every time a s
-  --workdir /root/ros2_ws/ \
-  -v .:/root/ros2_ws/src/inorbit_republisher \
+  --workdir /root/ROS 2_ws/ \
+  -v .:/root/ROS 2_ws/src/inorbit_republisher \
```
And here

FlorGrosso commented on inorbit_republisher/README.md:
```diff
@@ -197,7 +197,7 @@ colcon build --packages-select inorbit_republisher --symlink-install
 source install/local_setup.bash
 # Using the launch file under the 'launch' directory
-ros2 launch inorbit_republisher example.launch.xml
+ROS 2 launch inorbit_republisher example.launch.xml
```
```suggestion
ros2 launch inorbit_republisher example.launch.xml
```

FlorGrosso commented on inorbit_republisher/README.md:
````diff
@@ -173,21 +173,21 @@ These values will be published as latched and delivered only once every time a s
 
 ```bash
-cd ~/ros2_ws
+cd ~/ROS 2_ws
````
```suggestion
cd ~/ros2_ws
```

IrinaTerebiznik committed:
Update README.md

Changed  every "ROS2"  that is not a command nor a URL to "ROS 2", as it is properly written that way.

IrinaTerebiznik commented:
Second commit should fix those errors
